get_hypothesis.py

```diff
@@ -0,0 +1,114 @@
+from transformers import logging, AutoConfig, AutoTokenizer, AutoModel
```

Comment:
`get_hypothesis.py` is a too general name for the module, because in this context it does not hint in any way that this is where bert-extractive-summarizer is used.